src/middleware/mfa-methods.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Response, Request } from "express";
+
+export function mfaMethodsMiddleware(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): void {
+  if (req.session.user.isPhoneNumberVerified) {
```

Comment:
Do we need to do this? Won't the call to method-management tell us what the users PRIMARY method is explicitly without us needing to infer it like this?

Reply:
yes, but right now, the call to the API doesn't exist, so this simulates it. 

If [Alfonso's PR](https://github.com/govuk-one-login/di-account-management-frontend/pull/1228) is merged in before, I will remove this middleware and use Alfonso's.

Reply:
Once https://github.com/govuk-one-login/di-account-management-frontend/pull/1228/files is merged in, I can remove this file and use the new middleware.